cypress/integration/my_patches.js

```diff
@@ -0,0 +1,31 @@
+/// <reference types="Cypress" />
+
+const MY_PATCHES_ROUTE =
+  "/my-patches";
+describe("my patches page", function() {
+  beforeEach(() => {
+    cy.server();
+    cy.login();
+  });
+
+  it("Should render with Show Commit Queue box checked when commitQueue not indicated in URL query param", () => {
+    cy.visit(MY_PATCHES_ROUTE);
+    cy.get(".cy-checkbox")
+      .find("input")
+      .should("be.checked");
+  });
+
+  it("Should render with Show Commit Queue box checked when commitQueue is false in URL query param", () => {
```

Comment:
i think this should be "Should render with Show Commit Queue box UNchecked when commitQueue is false in URL query param"